Source/ARTAuth.m

```diff
@@ -184,22 +185,23 @@ - (void)requestToken:(ARTTokenParams *)tokenParams withOptions:(ARTAuthOptions *
         }];
     } else {
         void (^tokenDetailsFactory)(ARTTokenParams *, void(^)(ARTTokenDetails *__art_nullable, NSError *__art_nullable));
-        if (mergedOptions.authCallback) {
+        if (replacedOptions.authCallback) {
             tokenDetailsFactory = ^(ARTTokenParams *tokenParams, void(^callback)(ARTTokenDetails *__art_nullable, NSError *__art_nullable)) {
-                mergedOptions.authCallback(tokenParams, ^(id<ARTTokenDetailsCompatible> tokenDetailsCompat, NSError *error) {
+                replacedOptions.authCallback(tokenParams, ^(id<ARTTokenDetailsCompatible> tokenDetailsCompat, NSError *error) {
                     [tokenDetailsCompat toTokenDetails:self callback:callback];
                 });
             };
             [self.logger debug:@"RS:%p ARTAuth: using authCallback", _rest];
         } else {
             tokenDetailsFactory = ^(ARTTokenParams *tokenParams, void(^callback)(ARTTokenDetails *__art_nullable, NSError *__art_nullable)) {
                 // Create a TokenRequest and execute it
-                [self createTokenRequest:currentTokenParams options:mergedOptions callback:^(ARTTokenRequest *tokenRequest, NSError *error) {
+                [self createTokenRequest:currentTokenParams options:replacedOptions callback:^(ARTTokenRequest *tokenRequest, NSError *error) {
                     if (error) {
                         callback(nil, error);
                     } else {
                         [self executeTokenRequest:tokenRequest callback:callback];
                     }
+                    _isOnlyForceTrue = NO;
```

Comment:
This change doesn't have any effect, does it? The only other place where `_isOnlyForceTrue` is used is in `authorise`, and  there it's always being set before being read.